export declare function catchAndPrintErrors(fn: () => Promise<void>): () => Promise<void>;
